core/store/models/log_events.go

```diff
@@ -482,6 +487,14 @@ func (parseRunLog20190207withoutIndexes) parseJSON(log eth.Log) (JSON, error) {
 	return js.Add("functionSelector", OracleFulfillmentFunctionID20190128withoutCast)
 }
 
+func bytesToUintBigEndian(b []byte) (res uint64) {
```

Comment:
I believe we already have code that does this in the codebase.

Reply:
This is pulled straight from go-ethereum's codebase.

Here: https://github.com/ethereum/go-ethereum/blob/master/whisper/whisperv6/whisper.go#L1021

Reply:
Why yes.